stubs/requests/requests/exceptions.pyi

```diff
@@ -1,18 +1,21 @@
+from typing import Any
+
 from urllib3.exceptions import HTTPError as BaseHTTPError
 
 from .models import Request, Response
 
 class RequestException(OSError):
     response: Response | None
     request: Request | None
-    def __init__(self, *args, **kwargs) -> None: ...
+    def __init__(self, *args: Any, request: Request | None = ..., response: Response | None = ...) -> None: ...
```

Comment:
```suggestion
    def __init__(self, *args: object, request: Request | None = ..., response: Response | None = ...) -> None: ...
```
Consistent with how we annotate `BaseException.__init__`.

Reply:
Good point, changed.